src/analyses/libraryFunctions.ml

```diff
@@ -50,6 +86,17 @@ let gcc_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("__builtin_unreachable", special' [] @@ fun () -> if get_bool "sem.builtin_unreachable.dead_code" then Abort else Unknown); (* https://github.com/sosy-lab/sv-benchmarks/issues/1296 *)
     ("__assert_rtn", special [drop "func" [r]; drop "file" [r]; drop "line" []; drop "exp" [r]] @@ Abort); (* gcc's built-in assert *)
     ("__builtin_return_address", unknown [drop "level" []]);
+    ("error", unknown ((drop "status" []):: (drop "errnum" []) :: (drop "format" [w]) :: (VarArgs (drop' [r]))));
+    ("fputs_unlocked", unknown [drop "s" [r]; drop "stream" [w]]);
```

Comment:
Why is this `unlocked` function here, while others are above in POSIX?

Reply:
Iirc, this one is a GCC thing, and not in the standard.

Reply:
Right, they're all on the same man page, but I didn't look at the `CONFORMING TO` section carefully enough.
I think the appropriate thing to do with this and the similar ones below is to add a new spec list  for glibc, for things that glibc provides, but are not in C/POSIX standard nor GCC builtins.

Reply:
It's a non-standard extension of POSIX that is supported by `glibc`, see the [Conforming To](https://linux.die.net/man/3/fputs_unlocked) here.

Reply:
It's a non-standard extension of POSIX that is supported by `glibc`, see the [Conforming To](https://linux.die.net/man/3/fputs_unlocked) here.